src/d_event.h

```diff
@@ -73,7 +74,7 @@ typedef struct
     evtype_t type;
 
     // Event-specific data; see the descriptions given above.
-    int data1, data2, data3, data4, data5;
+    int data1, data2, data3, data4, data5, data6;
```

Comment:
Nothing in this PR appears to actually use this new information, so it seems incomplete.